src/plugins/discover/public/application/embeddable/search_embeddable.ts

```diff
@@ -221,6 +221,7 @@ export class SearchEmbeddable
       if (!searchScope.columns) {
         return;
       }
+      throw new Error('test error - searchscope');
       indexPattern.popularizeField(columnName, 1);
```

Comment:
I'm not aware of a way to add columns in the embeddable? while we may keep this code unless we're 100% sure, I don't think we should keep popularize